src/App.vue

```diff
@@ -1,38 +1,136 @@
 <script setup lang="ts">
+
   import Header from './components/Header.vue'
+
+  import CardSingle from './components/CardSingle.vue'
+  import CardGroup from './components/CardGroup.vue'
+  import CardSingleSource from './components/CardSingleSource.vue'
+  import CardSingleSocial from './components/CardSingleSocial.vue'
+
   import TopStory from './components/TopStory.vue'
   import MultipleHeadlines from './components/MultipleHeadlines.vue'
   import SingleVideo from './components/SingleVideo.vue'
   import SingleSourceRow from './components/SingleSourceRow.vue'
   import MayHaveMissed from './components/MayHaveMissed.vue'
   import Mastodon from './components/Mastodon.vue'
+
   import { getRandomNumbersInRange } from './components/helpers/sprinkle_getRandomNumbersInRange'
   
   var row1: number = getRandomNumbersInRange(1,0,24)
   var row2: number = getRandomNumbersInRange(1,0,24)
   var row3: number = getRandomNumbersInRange(1,0,24)
   var row4: number = getRandomNumbersInRange(1,0,24)
+
 </script>
 
 <template>
   <Header msg="scoopy" />
   <section class="module-row">
 
-    <TopStory headline="" body="" />
-    <MultipleHeadlines :count=4 />
-    <MultipleHeadlines :count=4 />
-    <MayHaveMissed />
-    <Mastodon :count=3 />
-    <SingleSourceRow :source=0 :count=50 />
-    <SingleVideo />
-    <SingleVideo />
-    <SingleSourceRow :source=row1 :count=50 />
-    <SingleSourceRow :source=row2 :count=50 />
-    <MultipleHeadlines :count=4 />
-    <MayHaveMissed />
-    <MultipleHeadlines :count=4 />
-    <SingleSourceRow :source=row3 :count=50 />
-    <SingleSourceRow :source=row4 :count=50 />
+    <CardSingle
+      card_title="Top Story"
+      card_description="A single card. Only has one bit of content (article) inside, and adds a card footer."
+      >
+      <TopStory />
+    </CardSingle>
+
+    <CardGroup
+      card_title="Just the Headlines"
+      card_description="A card group. Contains multiple bits of content (articles) inside, NO footer."
+      >
+      <MultipleHeadlines :count=4 />
+    </CardGroup>
+
+    <CardGroup
+      card_title="Just the Headlines"
+      card_description=""
+      >
+      <MultipleHeadlines :count=4 />
+    </CardGroup>
+
+    <CardSingle
+      card_title="You May Have Missed"
+      card_description="I'm not entirely sure what this feed... is."
```

Comment:
the idea of the may have missed is to bubble up specifically stale content. so like 1 week old or something like that. in the LONG term it would be old content that we know you havent clicked before

Reply:
What's it doing now? At the very least, want to pull in more items? It looks so wonky surrounded by tall cards.

Reply:
right now its basically just another design treatment of the top story card. Yeah maybe it should have 2  items within it for now. eventually our unfurl will hopefully be improved to where often its a long amount of content for a single item tho